CMakeLists.txt

```diff
@@ -22,21 +22,17 @@ include(Common)
 # Make Brayns optional when OSPRay is not available and:
 # - building as a subproject
 # - building on Ubuntu Jenkins VM
+set(OSPRAY_REQUIRED)
 if(NOT ${CMAKE_SOURCE_DIR} STREQUAL ${PROJECT_SOURCE_DIR} OR
         (TRAVIS AND LSB_DISTRIBUTOR_ID MATCHES "Ubuntu"))
-  common_find_package(OSPRay)
-  if(NOT OSPRAY_FOUND)
-      message(WARNING "[Brayns] OSPRay not found, the project will not be built")
-    include(CommonPackageConfig)
-    return()
-  endif()
+  set(OSPRAY_REQUIRED REQUIRED)
```

Comment:
what? Why should ospray be required for brayns? it also works with optix and/or livre. Why this behavioral change?